game/src/info/mod.rs

```diff
@@ -571,6 +572,15 @@ impl InfoPanel {
                             ctx, app,
                         ))),
                     )
+                } else if let Some(x) = action.strip_prefix("routes across Intersection #") {
```

Comment:
This is increasingly messy, but I'm not sure of a nice workaround yet. Info panels plumb around `Details`, which have a way of mapping button clicks to opening a different info panel tab (`hyperlinks`), warping to an object (`warpers`), and jumping to a trip at some time (`time_warpers`). Maybe the refactor would be to map button clicks to callbacks that'd produce `(close the info panel, optional transition)`